message_definitions/v1.0/messages.xml

```diff
@@ -2647,6 +2647,12 @@
     <field name="data" type="uint8[]"/>
   </message>
 
+ <message name="EXTERNAL_MAGNETO_RAW" id="154" link="forwarded">
```

Comment:
Oh I just copy-pasted the HITL_INFRARED message and changed it, but half way down the class, the indentation style changes.